### PR TITLE
Add keydown support

### PR DESCRIPTION
**Pull request by selimata**
Add keydown support for next(), previous(), and see more events.

### CODE REVIEW

File: src/components/SeeMore.tsx [hunk 1]
```diff
@@ -1,7 +1,25 @@
...
+            document.removeEventListener("keydown", handleKeyDown);
+        }
+    }, [])
+
+    const handleKeyDown = (e: KeyboardEvent) => {
+        if (e.key === 'ArrowUp') {
+            e.preventDefault()
+            props.toggleMore(true)
+        }
+        else if (e.key === 'ArrowDown') {
```
Comment (mohitk05) [inline src/components/SeeMore.tsx:17]: Can we also add `Esc` for the escape key here? Would make sense in addition to the ArrowDown.

Comment (selimata) [reply src/components/SeeMore.tsx]: I also added support for Esc key.

File: src/components/Container.tsx [hunk 2]
```diff
@@ -33,6 +33,22 @@ export default function () {
         }
     }, [isPaused])
 
+    useEffect(() => {
+        document.addEventListener("keydown", handleKeyDown);
```
Comment (mohitk05) [inline src/components/Container.tsx:37]: Let's make these events conditional and switchable via a prop called `keyboardNavigation` (boolean) for the root import as not everyone might want their keys to respond to these changes.
You can add this prop to `GlobalCtx` type and include it inside the context object in the root `index.tsx`. Then you can import it from the `useContext` hook. For `Container.tsx` it is already imported, you would have to mention it in the de-constructor. For `SeeMore.tsx`, you'd have to import it and use as in Container.tsx. Attach the event listeners only if the prop is `true`. Let me know if you need any help here.

**Comment by mohitk05** [comment]
@selimata Thanks for sending in a PR and adding this feature! I have left one more comment on the PR, do check and let me know here if you need any help. Cheers!

File: src/components/Container.tsx (hunk 2 above)
Comment (selimata) [reply src/components/Container.tsx]: I also added new things.

FYI: I haven't made any changes to the README.md file, but If you want me to update it, I can update it. This is my first contribution to a project so this reason I didn't want to make changes without asking. Thank you for your understanding.

**Comment by mohitk05** [comment]
Thank you again!

**Pull request merged**